core/src/main/java/org/kiwix/kiwixmobile/core/page/viewmodel/effects/OpenPage.kt

```diff
@@ -30,6 +31,7 @@ data class OpenPage(
 ) : SideEffect<Unit> {
   override fun invokeWith(activity: AppCompatActivity) {
     activity as CoreMainActivity
+    activity.popNavigationBackstack()
```

Comment:
I would expect to see a unit test update for this added functionality in a SideEffect

Reply:
@Frans-Lukas i am correct in thinking you won't have availability until next weekend? I can finish this up if needed

Reply:
This seems like a quick fix. If you have the time, go ahead, if not, tell me and I'll do it tonight or tomorrow.

Reply:
I am eager to release so I'll add the test